### DISCUSSION TITLE
Enhancement: Key rebinding

### DISCUSSION THREAD

**Discussion by Shady-Heretic** [FunkinCrew/Funkin#1603] [Ideas]
I would really like to rebind my keys as my style for playing rhythm games is pretty unique so it makes it hard for me to play the game with the current static binds.

**Comment by GStudiosX2** [comment]
He wouldn't add this anyway you should be able to edit controls.txt it has all the controls

**Comment by GStudiosX2** [comment]
i mean there might be a options menu not sure ik there is one but its unfinished
